website/docs/foundations/elevation/partials/guidelines/guidelines.md

```diff
@@ -1,4 +1,20 @@
-## Layers
+## Usage
 
-![Elevation design usage from Figma](/assets/images/elevation-design-usage.png)
+Use elevation:
+
+- to surface interactivity
+- to elevate and group key content on the page
+- to convey natural page hierarchy and modality
+
+## Levels of elevation
+
+| Level  |  Elevation  |   |  Surface  |    |  Object type  |
+|--------|-------------|---|-----------|----|---------------|
+| **Overlay**  |  <div class="hds-elevation-overlay" style="width: 40px; height: 40px; border-radius: 3px;"><Doc::Placeholder @width="40" @height="40" @background="transparent" /></div>  |   |  <div class="hds-surface-overlay" style="width: 40px; height: 40px; border-radius: 3px;"><Doc::Placeholder @width="40" @height="40" @background="transparent" /></div>  |  Modals  |    |
```

Comment:
this is totally unmaintainable. let's replace it with an image.

Reply:
The text in an image cannot be searched, and I don't imagine we'll be changing our elevation values often, so I personally don't see much risk in keeping this.

Reply:
I would at least probably do the table all in HTML vs. a combo of markup with nested HTML and also create a doc component for the repeated examples vs. all the inline styles. That would help readability of the code a lot I think.

Reply:
Jory's idea of using images in a table is a good idea too I think, then you wouldn't need to create a special component like I suggested.

Btw, to get the HTML markup for the table, you could use dev tools to view the rendered code of your page, then copy and paste it. Then edit it to add images for the examples instead of divs with inline styles.

Reply:
> The text in an image cannot be searched, and I don't imagine we'll be changing our elevation values often, so I personally don't see much risk in keeping this.
> Jory's idea of using images in a table is a good idea too I think, then you wouldn't need to create a special component like I suggested.
> Btw, to get the HTML markup for the table, you could use dev tools to view the rendered code of your page, then copy and paste it.

@heatherlarsen @KristinLBradley @jorytindall + @Dhaulagiri 

When we started the new website project, we discussed the pros and cons of the different options that were available, and one of the clear limitations of using markdown was that it could become complex to achieve specific layouts. 
This is one of these cases. The fact that technically it can be done, doesn't mean it should be done.
I know an image is not ideal, but the code above, even if we use single images, is beyond an acceptable level of maintenance (I know you think we will not change it often that code, but we will change other code, eg. the markdown table parser, or the placeholder, or something else, and then things can become messy: just one character off, and the entire rendering blows up, and debugging and finding the error becomes a pain).
Please, use an image in this specific case.

Reply:
After speaking with @Dhaulagiri, we decided to move forward with an image for launch. I've updated the image to use styles from the website to fit more seamlessly within the documentation.